src/app/public/src/modules/wrapper/wrapper.component.ts

```diff
@@ -74,6 +78,15 @@ export class StacheWrapperComponent implements OnInit, AfterContentInit, OnDestr
     this.destroyPageAnchorSubscriptions();
   }
 
+  private checkShowEditSetting(): boolean {
+    if (
+      this.config.skyux.appSettings
+      && this.config.skyux.appSettings.stache
+    ) {
+      return this.config.skyux.appSettings.stache.showEdit || false;
+    }
```

Comment:
There's a chance that this method won't return a boolean. Might need to rework the logic so that every branch returns true or false.

Reply:
This will always return a boolean. If showEdit is undefined, which is a falsey value, it'll return false, which is the desired behavior. I could add another check and handle the undefined explicitly, but I don't see a need to.

Reply:
I have a better way to handle this now, so this won't be an issue any more.

Reply:
:clinking_glasses: